docs/csharp/language-reference/keywords/enum.md

```diff
@@ -9,85 +9,93 @@ helpviewer_keywords:
-  
-```csharp  
-enum Day : byte {Sat=1, Sun, Mon, Tue, Wed, Thu, Fri};  
-```  
-  
- The approved types for an enum are [byte](../../../csharp/language-reference/keywords/byte.md), [sbyte](../../../csharp/language-reference/keywords/sbyte.md), [short](../../../csharp/language-reference/keywords/short.md), [ushort](../../../csharp/language-reference/keywords/ushort.md), [int](../../../csharp/language-reference/keywords/int.md), [uint](../../../csharp/language-reference/keywords/uint.md), [long](../../../csharp/language-reference/keywords/long.md), or [ulong](../../../csharp/language-reference/keywords/ulong.md).  
-  
- A variable of type `Day` can be assigned any value in the range of the underlying type; the values are not limited to the named constants.  
-  
- The default value of an `enum E` is the value produced by the expression `(E)0`.  
-  
+
+Usually it is best to define an enum directly within a namespace so that all classes in the namespace can access it with equal convenience. However, an enum can also be nested within a class or struct.
+
+By default, the first enumerator has the value 0, and the value of each successive enumerator is increased by 1. For example, in the following enumeration, `Sat` is `0`, `Sun` is `1`, `Mon` is `2`, and so forth.
+
+```csharp
+enum Day {Sat, Sun, Mon, Tue, Wed, Thu, Fri};
+```
+
+Enumerators can use initializers to override the default values, as shown in the following example.
+
+```csharp
+enum Day {Sat=1, Sun, Mon, Tue, Wed, Thu, Fri};
+```
+
+In this enumeration, the sequence of elements is forced to start from `1` instead of `0`. However, including a constant that has the value of 0 is recommended. For more information, see [Enumeration Types](../../../csharp/programming-guide/enumeration-types.md).
+
+Every enumeration type has an underlying type, which can be any integral type except [char](../../../csharp/language-reference/keywords/char.md). The default underlying type of enumeration elements is [int](../../../csharp/language-reference/keywords/int.md). To declare an enum of another integral type, such as [byte](../../../csharp/language-reference/keywords/byte.md), use a colon after the identifier followed by the type, as shown in the following example.
+
+```csharp
+enum Day : byte {Sat=1, Sun, Mon, Tue, Wed, Thu, Fri};
+```
+
+The approved types for an enum are [byte](../../../csharp/language-reference/keywords/byte.md), [sbyte](../../../csharp/language-reference/keywords/sbyte.md), [short](../../../csharp/language-reference/keywords/short.md), [ushort](../../../csharp/language-reference/keywords/ushort.md), [int](../../../csharp/language-reference/keywords/int.md), [uint](../../../csharp/language-reference/keywords/uint.md), [long](../../../csharp/language-reference/keywords/long.md), or [ulong](../../../csharp/language-reference/keywords/ulong.md).
```

Comment:
not related to this PR:  I've been shrinking the paths for simplicity (../../../csharp/language-reference/keywords/byte.md) becomes (byte.md)